roles/test_alerts/tasks/test_create_an_alert.yml

```diff
@@ -24,10 +24,23 @@
                 expr: rate(sg_total_collectd_msg_received_count[1m]) == 0
       EOF
   changed_when: false
+  register: cmd_output
+  failed_when: cmd_output.rc != 0
+
+
+- name: "Show the command output"
+  ansible.builtin.debug:
+    var: cmd_output
```

Comment:
Do you want the whole returned object, or the stdout?

You can return just the stdout using

cmd_output.stdout
OR
cmd_output.stdout_lines